doc/faq.rst

```diff
@@ -67,6 +67,25 @@ Miscellaneous
    See also :func:`create_adof_conn()
    <sfepy.discrete.variables.create_adof_conn>`.
 
+#. Visualization of various FEM-related information.
+
+   - Quadrature rules::
+
+       python3 script/plot_quadratures.py
+
+   - Facet orientations::
+
+       python3 sfepy/postprocess/plot_facets.py
```

Comment:
Add::

```
import sys
sys.path.append('.')
```
to `plot_facets.py` script.

Reply:
This is a regular module, so I would rather not meddle with sys.path on its import. Instead, I explained PYTHONPATH setting in the FAQ.

Reply:
OK